Meadowlark-js/backends/meadowlark-mongodb-backend/src/repository/Db.ts

```diff
@@ -8,7 +8,7 @@ import { Logger } from '@edfi//meadowlark-core';
 import { MeadowlarkDocument } from '../model/MeadowlarkDocument';
 
 const MONGO_URL_DEFAULT = 'mongodb://mongo1:27017,mongo2:27018,mongo3:27019/?replicaSet=rs0';
-export const DATABASE_NAME = 'meadowlark';
+export const DATABASE_NAME = process.env.MEADOWLARK_DATABASE_NAME || 'meadowlark';
```

Comment:
Something to keep in mind is that more configurability means more testing is needed. For example, the Debezium connector uses the database name as part of the Kafka topic name.

Reply:
I do think its important to let this vary. In particular I was motivated by not wanting to destroy my database every time I run system or integration tests.